src/jit/importer.cpp

```diff
@@ -1661,10 +1661,7 @@ GenTreePtr          Compiler::impLookupToTree(CORINFO_LOOKUP *pLookup, unsigned
         // Don't import runtime lookups when inlining
         // Inlining has to be aborted in such a case
         JITLOG((LL_INFO1000000, INLINER_FAILED "Cannot inline complicated handle access\n"));
-
-        JitInlineResult result(INLINE_FAIL, impInlineInfo->inlineCandidateInfo->ilCallerHandle,
-                               info.compMethodHnd, "Cannot inline generic dictionary lookup");
-        compSetInlineResult(result);  // Only fail for this inline attempt.
+        compInlineResult->setFailure("Cannot inline generic dictionary lookup");
         return NULL;
```

Comment:
Since your changes are in close proximity, you might consider changing these NULLs to nullptrs.